### PR TITLE
2415 acceptance tests using api

### PR DESCRIPTION
**Pull request by LukeLoze1**
Start using the support tools API to setup surveys, collexs, WOCs,  load samples and so on.

Test with master branches of everything else

**Fulfilment is 'still' using the database this will go on a new ticket possibly.  Needs discussion

### CODE REVIEW

**Review by NickGrantONS — APPROVED** [review]
Works like a dream. Nice work 👍

File: acceptance_tests/utilities/wave_of_contact_helper.py [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
+        'collectionExercise':  'collectionExercises/' + context.collex_id
+    }
 
-        wave_of_contact_query = """insert into casev3.wave_of_contact(id, classifiers,has_triggered, pack_code,
-        print_supplier,template,trigger_date_time
-        ,type,collection_exercise_id) values (%s, %s, 'f', %s, 'SUPPLIER_A',%s,%s,'PRINT',%s)"""
-        wave_of_contact_vars = (context.woc_uuid, context.classifiers, context.pack_code, context.template,
-                                trigger_date_time, context.collex_id)
-        cur.execute(wave_of_contact_query, vars=wave_of_contact_vars)
+    response = requests.post(url, auth=Config.BASIC_AUTH, json=body)
```
Comment (AdamHawtin) [inline acceptance_tests/utilities/wave_of_contact_helper.py:33]: I don't think the basic auth is needed here

File: config.py [hunk 2]
```diff
@@ -37,6 +35,14 @@ class Config:
     EXCEPTIONMANAGER_CONNECTION_PORT = os.getenv('EXCEPTIONMANAGER_CONNECTION_PORT', '8666')
     EXCEPTION_MANAGER_URL = f'http://{EXCEPTIONMANAGER_CONNECTION_HOST}:{EXCEPTIONMANAGER_CONNECTION_PORT}'
 
+    SECURITY_USER_NAME = os.getenv('SECURITY_USER_NAME', 'admin')
+    SECURITY_USER_PASSWORD = os.getenv('SECURITY_USER_PASSWORD', 'secret')
+    BASIC_AUTH = (SECURITY_USER_NAME, SECURITY_USER_PASSWORD)
```
Comment (AdamHawtin) [inline config.py:40]: I don't think basic auth is needed

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
👍 All passing, but I don't think the basic auth is doing anything and can be removed.

File: acceptance_tests/utilities/wave_of_contact_helper.py (hunk 1 above)
Comment (LukeLoze1) [reply acceptance_tests/utilities/wave_of_contact_helper.py]: removed

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**